hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -508,6 +508,16 @@ public long revokeCertificates(List<String> certIds, int reason,
     }
   }
 
+  @Override
+  public List<String> clearExpiredCertificates() throws IOException {
+    List<String> pemEncodedCerts = new ArrayList<>();
```

Comment:
I would suggest add the isAdmin check here. Here is an example.

 storageContainerManager.checkAdminAccess(getRpcRemoteUser(), false);